demos/imagenet_classification_demo/README.md

```diff
@@ -0,0 +1,122 @@
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Running the application with the <code>-h</code> option yields the following usage message:
+```sh
+./imagenet_classification_demo -h
+
+imagenet_classification_demo [OPTION]
+Options:
+
+    -h                        Print a usage message.
+    -i "<path>"               Required. Path to a folder with images or path to an image files: a .ubyte file for LeNet and a .bmp file for the other networks.
+    -m "<path>"               Required. Path to an .xml file with a trained model.
+      -l "<absolute_path>"    Required for CPU custom layers.Absolute path to a shared library with the kernels implementation.
+          Or
+      -c "<absolute_path>"    Required for GPU custom kernels. Absolute path to the .xml file with kernels description.
+    -labels "<path>"          Required. Path to .txt file with imagenet labels.
+    -gt "<path>"              Optional. Path to .txt file with image classes.
+    -d "<device>"             Optional. Specify the target device to infer on (the list of available devices is shown below). Default value is CPU. Sample will look for a suitable plugin for device specified.
+    -b "<integer>"            Optional. Specify batch to infer. Default value is 1.
+    -nthreads "<integer>"     Optional. Specify count of threads.
+    -nstreams "<integer>"     Optional. Specify count of streams.
+    -nireq "<integer>"        Optional. Number of infer requests.
+    -nt "<integer>"           Optional. Number of top results. Default value is 5. Must be >= 1.
+    -res "<WxH>"              Optional. Set image grid resolution in format WxH. Default value is 1920x1080.
+    -no_show                  Optional. Disable showing of processed images.
+    -time "<integer>"         Optional. Time in seconds to execute program. Default is -1 (infinite time).
+    -u                        Optional. List of monitors to show initially.
+```
+
+The number of infer requests can be set by "-nireq" flag.
+* small nireq: low FPS (device cores aren't fully loaded) and low latency (images don't wait for long before being sent
+for inference).
+* nireq = number of cores: high FPS (full device utilization), median latency (images wait for some time). Considered optimal.
+* nireq > number of cores: high FPS (same as in the previous case), high latency (images wait for long time).
```

Comment:
The description shouldn't be based on a number of cores because it is harder to apply that term for some types of devices.
I would write about the idea of using InferRequests as buffers, so the larger -nireq the bigger the buffer and bigger the latency, but it becomes easier to parallelize the jobs.

As for gaining FPS, this is not about nireq compared to cores, nireq should slightly exceed nstreams value summed over all used devices. (And in case of CPU nstreams should be equeal to number of cores.)

I also doubt about `nireq = number of cores: high FPS (full device utilization)` assuming CPU only is used. The pipeline needs an additional element to pre/post process while other images are inferred, so setting that exact number is sub-optimal.

I would also add a note somewhere about the demo reading all the images before an actual execution, which may cause the demo to seem doing nothing for a user.